dm/dm/master/server_test.go

```diff
@@ -573,6 +573,47 @@ func (t *testMaster) TestWaitOperationOkRightResult(c *check.C) {
 	}
 }
 
+func (t *testMaster) TestStopTaskWithExceptRight(c *check.C) {
+	taskName := "test-stop-task"
+	notExceptResp := &pb.QueryStatusResponse{
+		SubTaskStatus: []*pb.SubTaskStatus{
+			{
+				Name: "task1",
+				Status: &pb.SubTaskStatus_Sync{Sync: &pb.SyncStatus{
+					UnresolvedGroups: []*pb.ShardingGroup{{Target: "`db`.`tbl`", Unsynced: []string{"table1"}}},
+				}},
+			},
+		},
+	}
+	exceptResp := &pb.QueryStatusResponse{SubTaskStatus: []*pb.SubTaskStatus{
+		{
+			Name:   "task1",
+			Status: &pb.SubTaskStatus_Msg{Msg: common2.NoSubTaskMsg(taskName)},
```

Comment:
i am confused with this test, why we have so many task names?  `test-stop-task` ,`task1`, `task-unittest`

Reply:
they should be one name to imitate the real case.